apps/namegraph.dev/lib/utils.ts

```diff
@@ -1,6 +1,126 @@
 import { clsx, type ClassValue } from "clsx";
 import { twMerge } from "tailwind-merge";
+import {
+  NameGraphGroupedByCategoryResponse,
+  NameGraphGroupingCategory,
+  NameGraphSuggestion,
+} from "@namehash/namegraph-sdk/utils";
+import { createNameGraphClient } from "@namehash/namegraph-sdk";
 
 export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs));
 }
+
+interface LabelAndModeQueryParams {
```

Comment:
I'm confused here. It looks like we're smashing together two separate goals that should be seprated.

1. The definition of an interface associated with a NameGraph API query.
2. Specific constants that are sent in our implementation of this query within our example app.

As i understand it, (1) above should live in the SDK. (2) should live inside of this app, but it should not be achieved by defining a NEW interface. Instead it should be achieved by making reference to the interface defined in (1).